axis_backend/apps/audit/repositories/entity_change_repository.py

```
@@ -0,0 +1,101 @@
+        cutoff = timezone.now() - timezone.timedelta(days=days)
+        return self.get_queryset().filter(changed_at__gte=cutoff)
+
+    def search_changes(
+        self,
+        entity_type: Optional[str] = None,
+        entity_id: Optional[str] = None,
+        change_type: Optional[str] = None,
+        changed_by: Optional[str] = None,
+        start_date: Optional[datetime] = None,
+        end_date: Optional[datetime] = None,
+        is_active: Optional[bool] = None
```

Ruff (0.14.5)

Boolean-typed positional argument in function definition

(FBT001)